### PR TITLE
css-two-column-layout - Jimmy Vu

### PR DESCRIPTION
**Pull request by Jimmy-Vu**
Screen Shot 2021-08-23 at 9 52 33 AM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <img src="images/software.svg" alt="Picture of Africa"  class="info-images"/>
+        <div class="info-text">
+          <h3>INTUITIVE, SMART SOFTWARE</h3>
+          <p>We build products that transform how healthcare facilities run across Africa -
+            making them radically more efficient operationally and financially, a complete
+            turnaround.
+          </p>
+        </div>
+
+        <div class="column-one-tenth"></div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:32]: 😱 😱 😱 😱 😱 
You should _**never**_ use empty elements to create spacing. This is a job for `margin` or `padding`

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <div class="info-text">
+          <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+          <p>Our all-in-one electronic health records (EHR), hospital management (HMIS)
+            &amp; billing system helps streamline your facility,
+            reduce waste and iprove quality of care.
+          </p>
+        </div>
+      </section>
+
+      <div class="row-divider"></div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:44]: 😬

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <section class="row">
+        <img src="images/circumstance.svg" alt="Picture of revolving neurons" class="info-images" />
+        <div class="info-text">
+          <h3>FOR ANY CIRCUMSTANCE</h3>
+            <p>Our intuitive system is built around the peculiarities of your facility;
+               we support offline deployments, sync to the cloud and run on any device 24/7.
+          </p>
+        </div>
+
+        <div class="column-one-tenth"></div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:55]: 🤢

File: css-two-column-layout/index.html (hunk 2 above)
Comment (codingcodymiller) [reply css-two-column-layout/index.html:44]: `padding` or `margin`

File: css-two-column-layout/index.html (hunk 3 above)
Comment (codingcodymiller) [reply css-two-column-layout/index.html:55]: `padding or `margin`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! A few notes for you. Also, I see that you have that one column class you created to make space between elements, but none of your content in your `row` elements has any `column` classes to size or organize them. You should revise this to better follow the system we discussed in `css-layout-classes`

**Comment by Jimmy-Vu** [comment]
Screen Shot 2021-08-29 at 8 57 04 PM

**Review by codingcodymiller — APPROVED** [review]
Great work!!!

**Pull request merged**